test-apps/imodel-transformer/src/ElementUtils.ts

```diff
@@ -85,4 +88,49 @@ export namespace ElementUtils {
     }
     return elementIds;
   }
+
+  /** Generate and insert a ViewDefinition that views all models and all SpatialCategories.
+   * @param iModelDb The ViewDefinition will be inserted into this IModelDb.
+   * @param name The name (CodeValue) for the inserted ViewDefinition.
+   * @param makeDefault If `true` make the inserted ViewDefinition the default view.
+   * @returns The Id of the ViewDefinition that was found or inserted.
+   */
+  export function insertViewDefinition(iModelDb: IModelDb, name: string, makeDefault?: boolean): Id64String {
+    const definitionModelId = IModel.dictionaryId;
+    const viewCode = ViewDefinition.createCode(iModelDb, definitionModelId, name);
+    let viewId = iModelDb.elements.queryElementIdByCode(viewCode);
+    if (viewId === undefined) {
+      const modelSelectorId = ModelSelector.insert(iModelDb, definitionModelId, name, queryModelIds(iModelDb, SpatialModel.classFullName));
+      const categorySelectorId = CategorySelector.insert(iModelDb, definitionModelId, name, querySpatialCategoryIds(iModelDb));
+      const displayStyleId = DisplayStyle3d.insert(iModelDb, definitionModelId, name);
+      viewId = OrthographicViewDefinition.insert(iModelDb, definitionModelId, name, modelSelectorId, categorySelectorId, displayStyleId, iModelDb.projectExtents, StandardViewIndex.Iso);

```

Comment:
Why orthographic??

Reply:
No good reason. Is my latest change better?

Reply:
Yes. OrthographicViewDefinition is almost never what you want, all it does is forbid you from turning camera on.